tests/layer_tests/pytorch_tests/test_div.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import numpy as np
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class Testdiv(PytorchLayerTest):
+    def _prepare_input(self):
+        return (self.input_tensor, self.other_tensor)
+
+    def create_model(self, rounding_mode):
+
+        import torch
+
+        class aten_div(torch.nn.Module):
+            def __init__(self, rounding_mode):
+                super(aten_div, self).__init__()
+                self.rounding_mode = rounding_mode
+
+            def forward(self, input_tensor, other_tensor):
+                return torch.div(input_tensor, other_tensor, rounding_mode=self.rounding_mode)
+
+        ref_net = None
+
+        return aten_div(rounding_mode), ref_net, "aten::div"
+
+    @pytest.mark.parametrize('input_tensor', ([
+        np.random.randn(5).astype(np.float32),
+        np.random.randn(5, 5, 1).astype(np.float32),
+        np.random.randn(1, 1, 5, 5).astype(np.float32),
+    ]))
+    @pytest.mark.parametrize('other_tensor', ([
+        np.array([[0.5]]).astype(np.float32),
+        np.random.randn(5).astype(np.float32),
+        np.random.randn(5, 1).astype(np.float32),
+        np.random.randn(1, 5).astype(np.float32),
+    ]))
```

Comment:
float32 is tested only, it doesn't expose possible issues with our conversion when https://pytorch.org/docs/stable/tensor_attributes.html#type-promotion-doc is used. Actually it is relevant to other basic arithmetic ops according to the doc. I think we should implement tests for those case and let them fail. And implement more sophisticates support in conversion functions eventually.

Reply:
@slyalin I have added tests with not matching types which as expected failed.

Reply:
I believe initial task was extend op to support such behavior, not only tests

Reply:
another thing, it is TO DO in operation which also required to be fixes for claiming op supported:

>  // TODO: support mode "trunc", need to extend openvino opset for that 

so, why you add only tests?

Reply:
@bszmelcz agree with @eaidova, based on ONNX export you can use `Convert` to `int64` to support `trunc` divide

Reply:
Ok I will take care of that